README.rst

```diff
@@ -90,7 +90,7 @@ a ``make clean`` at the top-level first.)
 
 To get an optimized build of Python, ``configure --enable-optimizations``
 before you run ``make``.  This sets the default make targets up to enable
-Profile Guided Optimization (PGO) and may be used to auto-enable Link Time
+Profile Guided Optimization (PGO) may be used to auto-enable Link Time
```

Comment:
```
This sets the default make targets up to enable
Profile Guided Optimization (PGO). It may be used [...] the sections below.
```
What about this ?